### ISSUE TITLE
Only works with Enter

### ISSUE THREAD

**Issue by Qodestackr** [omeralpi/shadcn-phone-input#45]
Cannot Select a country with a mouse or in mobile touch unless you press "->"

**Comment by Desgue** [comment]
I am having the same issue. I cannot click on the countries with the mouse, but i can navigate them using the keyboard arrows and select using enter. 

I just updated the code and ran npm i again to make sure I am using the latest version but the error continues.

**Comment by omeralpi** [comment]
> I am having the same issue. I cannot click on the countries with the mouse, but i can navigate them using the keyboard arrows and select using enter. 
> 
> 
> 
> I just updated the code and ran npm i again to make sure I am using the latest version but the error continues. 

Same issue #41

**Comment by Desgue** [comment]
I found the solution to my problem. 
First I updated all shadcn components needed for the phone input by using shadcn add. That solved the disabled look of the popover but I still had problems with the Popover behavior. I was still not able to use the mouse to interact. The problem was I was using the PhoneInput inside a Dialog. Whenever I used outside the Dialog it worked fine. 

After some searching i found this issue https://github.com/shadcn-ui/ui/issues/1511
Removing the Portal from the root inside popover.tsx fixed it for me. 

If you cant remove the Portal try setting the modal prop stated in the radix documentation
 https://www.radix-ui.com/primitives/docs/components/popover#api-reference
